pytest_splunk_addon/splunk.py

```diff
@@ -835,6 +835,16 @@ def update_recommended_fields(model, datasets, cim_version):
     return update_recommended_fields
 
 
+@pytest.mark.hookwrapper(hookwrapper=True)
```

Comment:
Looks like we're mixing two approaches here: `@pytest.mark.hookwrapper` which is deprecated, and the `hookwrapper=True` attribute that comes from the new annotation `@pytest.hookimpl`.
https://docs.pytest.org/en/7.4.x/deprecations.html#configuring-hook-specs-impls-using-markers 

```suggestion
@pytest.hookimpl(hookwrapper=True)
```

Reply:
Done